data/mods/gen9ssb/pokedex.ts

```diff
@@ -163,6 +163,14 @@ export const Pokedex: {[k: string]: ModdedSpeciesData} = {
 		abilities: {0: "Mime knows best"},
 	},
 
+	// Hydrostatics
+	pichuspikyeared: {
+		inherit: true,
+		baseStats: {hp: 35, atk: 55, def: 40, spa: 50, spd: 50, spe: 90},
+		abilities: {0: 'Hydrostatic Positivity'},
+		evos: ['Chandelure'], // Added for Eviolite compatibility
```

Comment:
```suggestion
		evos: ['Chandelure'], // Added for Eviolite compatibility
```
I'd rather you just edit Eviolite to get the same effect. See pre-IoA slowpoke-galar

Reply:
Did in https://github.com/smogon/ps-ssb/pull/87/commits/d83a9d3fa13806241306dcc227e2b897de2472d0